Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -77,14 +77,11 @@ final class PersonCell: UICollectionViewCell {
     }
     
     func configure(with person: Person) {
-        let df = DateFormatter()
-        df.dateFormat = "dd.MM.yyyy"
-        
         profileImageView.image = UIImage(named: "man-in-red-jacket-1681010")
         // ^Update to default place holder image when resource is added.
         // TO-DO: Use Kilo's ImageDownloader here to DL image
         nameLabel.text = person.fullName.uppercased()
-        dateOfIncidentLabel.text = df.string(from: person.date) 
+        dateOfIncidentLabel.text = person.date.localizedString(dateStyle: .short)
```

Comment:
The idea is that all date formatters are cached in `DateFormatterService` and reused. I see this one wasn't migrated to that yet, but now could be a good time to do it? Or can create a to-do ticket on Trello. Up to you :)

Reply:
The current implementation of caching formatters is done by using the given format as the key, yes? But the `DateFormatter.localizedString` method is a class function so I can use it without creating an instance of `DateFormatter` and passing a format. So I'm wondering how to fit that into the caching mechanism.

Reply:
yeah it's a bit confusing. I'd say there's no need to cache by key as it is done now - we will have only a handful of dates, so they can be just individual variables like `dateOfIncidentFormatter` etc each with their own configuration. 

Having a function like `localizedString...` is another approach which is also viable, though in this project date/time formatting has been agreed to be done via `DateFormatterService`, and it's good to stick to one way of doing it within a project 😊

Reply:
I agree. I moved the date formatting code to the `DateFormatterService`. Btw I was referring to the `[localizedString](https://developer.apple.com/documentation/foundation/dateformatter/1415241-localizedstring)` function in iOS itself earlier. I had also named my extension method the same name so that must have been confusing. Anyway, no more extension methods now.